servers/rendering/rasterizer_rd/rasterizer_effects_rd.cpp

```diff
@@ -1536,7 +1536,6 @@ RasterizerEffectsRD::RasterizerEffectsRD() {
 
 	default_sampler = RD::get_singleton()->sampler_create(sampler);
 
-	sampler.min_filter = RD::SAMPLER_FILTER_LINEAR;
```

Comment:
CC @clayjohn, looks a bit weird that not all members are set for each `sampler`.
Logic is preserved by the change though.

Reply:
Change seems fine to me, reduz does the same thing in other places anyway.